__tests__/app.test.js

```diff
@@ -41,4 +41,62 @@ describe("api", () => {
         });
     });
   });
+  // Responds with:
+
+  // a review object, which should have the following properties:
+
+  describe("/api/reviews/review_id", () => {
+    it("200 - responds with review object", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.length).toBe(1);
+          const review = body[0];
+          //Should be an Object
+
+          expect(review.hasOwnProperty("review_id", expect.any(Number))).toBe(
```

Comment:
Because you are expecting a specific review to be send back to you based on the endpoint, it would be good to test that the review with that id is being returned.